specification/hugr.md

```diff
@@ -788,9 +799,11 @@ Serialization section). It is also trivial to serialize these
 definitions in to the overall HUGR serialization format.
 
 Note the only required fields are `name` and `description`. `signature` is optional, but if present
-must have children `inputs` and `outputs`, each lists. The optional `misc` field is used for arbitrary
-YAML, which is read in as-is and passed to compiler passes and (if no `signature` is present) the
-`compute_signature` function; e.g. a pass can use the `basis` information to perform commutation.
+must have children `inputs` and `outputs`, each lists, and may have `resources`.
+
+The optional `misc` field is used for arbitrary YAML, which is read in as-is and passed to compiler
+ passes and (if no `signature` is present) the`compute_signature` function; e.g. a pass can use the `basis` information to perform commutation.
```

Comment:
I thought the conclusion of the discussion was that data in `misc` would not be used to compute signature?

Reply:
IIRC we said that if you wanted statically-known floats then you could put that in `misc` if you didn't want Static wires to every node. But I don't recall saying that `misc` couldn't affect signature. `misc` is a fairly generic mechanism to say "I want to reuse the same (Rust) code for different ops, I can express the difference with data" - I don't see why that shouldn't include/allow reusing Rust code that computes type signatures.